xapian-letor/api/featurevector_internal.cc

```diff
@@ -117,3 +117,31 @@ FeatureVector::Internal::get_nonzero_num(){
     //cout << "nonzero: "<< nonzero <<endl;
     return nonzero;
 }
+
+void
+FeatureVector::Internal::normalise() {
```

Comment:
Its intended use was to normalise each feature's values across the set of feature vectors contained in a `RankList` element. I did not realise this while removing `RankList` and blindly converted it to work on individual FeatureVectors. 

I will re-implement it as a utility function that normalises the complete `vector<FeatureVector>` returned by `create_vec_fvec` (or by `create_feature_vectors` method, after renaming).

Thanks for pointing this out.